src/shared/utils/build-get-iso2-from-name.js

```diff
@@ -0,0 +1,13 @@
+const assert = require('assert')
+const iso2 = require('country-levels/iso2.json')
+
+const buildGetIso2FromName = ({ country, specialCases }) => {
```

Comment:
@hyperknot This seems pretty handy, will hopefully remove most of those manual ID maps. Leveraging your awesome country levels repo

Reply:
Not sure what this one does by its name, can you elaborate?

Reply:
See above comment: this kind of utility cannot live in src/shared. It will push us over payload limits and slow everything down enormously. This needs to be factored into a specific event or worked around another way, unfortunately.

Reply:
Yup, will try to work with @hyperknot to get something cut-down to be usable within the lambda.